shared/build.gradle.kts

```diff
@@ -332,6 +332,9 @@ task<CycloneDxBomTransformTask>("bomIosSwiftPM") {
     inputPath = layout.buildDirectory.file("boms/bom-ios-swiftpm-raw.json")
     outputPath = layout.buildDirectory.file("boms/bom-ios-swiftpm.json")
     transform = {
+        // the cdxgen metadata gets emitted as the wrong kind of element after merging and breaks
+        // the parser, or something like that
+        metadata.tools = emptyList()
```

Comment:
I'm glad that we just rolled back #624 instead of chasing this down at the time.